eth/backend.go

```diff
@@ -337,6 +342,11 @@ func (s *Ethereum) APIs() []rpc.API {
 			Version:   "1.0",
 			Service:   s.netRPCService,
 			Public:    true,
+		}, {
+			Namespace: "eth2",
```

Comment:
See https://notes.ethereum.org/@n0ble/rayonism-the-merge-spec
This was renamed to "consensus". @mkalinin

Reply:
The preference is to use `consensus` name for this namespace. Should not be difficult to update, right?